### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by VezmenovDanil**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,159 +1,169 @@
...
-      alert("Finish editing the task and try again!");
+  document.addEventListener('keydown', (event) => {
+    let newTaskText = document.getElementById('newTaskText').value;
+    newTaskText = newTaskText.replace(/\s/g, '');
+    if (event.key === 'Enter' && counter === 1) {
+      // eslint-disable-next-line no-alert
+      alert('Finish editing the task and try again!');
     }
-    if (event.key == "Enter" && newTaskText != "" && counter === 0) {
+    if (event.key === 'Enter' && newTaskText !== '' && counter === 0) {
```
Comment (usatovaDasha) [inline script.js:166]: && !newTaskText  && !counter

File: style.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 body {
   margin: 0;
   padding: 0;
-  background-image: url(mainTheme.jpg);
+  background-image: url(picture/mainTheme.jpg);
 }
```
Comment (usatovaDasha) [inline style.css:5]: добавьте пустые строки между блоками

File: style.css [hunk 3]
```diff
@@ -141,6 +142,7 @@ body {
   margin: auto;
 }
 .task-checkbox {
+  color:black;
```
Comment (usatovaDasha) [inline style.css:145]: check spaces

File: script.js (hunk 1 above)
Comment (VezmenovDanil) [reply script.js:166]: Dear Daria, your version of the code did not work correctly, because I didn’t use it.

File: script.js [hunk 4]
```diff
@@ -143,27 +134,28 @@ window.onload = function onload() {
...
+      const tempNewText = tempLi.querySelector('.editor').value;
       const originalTask = getTaskById(tempTaskId);
       originalTask.taskText = tempNewText;
       counter = 0;
       render();
     }
     if (target.id === 'clearData') {
       localStorage.clear();
-      location.reload();
+      taskStorage.splice(0, taskStorage.length);
```
Comment (usatovaDasha) [inline script.js:146]: taskStorage = [];

File: script.js [hunk 5]
```diff
@@ -102,11 +91,13 @@ window.onload = function onload() {
         tempParentHolder.classList.remove('done-task-style');
       }
       render();
```
Comment (usatovaDasha) [inline script.js:93]: render last

File: style.css [hunk 6]
```diff
@@ -141,90 +159,93 @@ body {
...
+.display-none {
   display: none;
 }
 
+@media screen and (max-width: 1050px) {
+  .add-task-panel {
+    display: flex;
+    justify-content: space-around;
+    width: 95%;
+  }
```
Comment (usatovaDasha) [inline style.css:215]: empty line

File: style.css [hunk 7]
```diff
@@ -141,90 +159,93 @@ body {
...
+  .add-newtask-but {
+    width: 20%;
+    font-size: 1em;
+    padding: 0px;
+  }
+}
 
+@media screen and (max-width: 682px) {
```
Comment (usatovaDasha) [inline style.css:223]: add empty lines

**Pull request merged**